portfolio/src/main/webapp/index.html

```diff
@@ -75,6 +75,102 @@ <h1 id="name-heading">About Me</h1>
                         </button>
                     </td>
                 </tr> 
+                <tr>
+                    <td id="search-form-box" colspan="6">
+                        <input type="text" name="search" id="search" value="" tabindex="1" placeholder="Search">
+                    </td>
+                    <td id="clear-search-box" >
+                        <button onclick="onClearSearch()">X</button>
+                    </td>
+                </tr>
+               <tr>
+                    <td colspan="7" id="select-language-box">
+                        <select name="select-language" id="select-language" 
+                           onchange="displayCommentSection('none')">
+                           <option value="none">Select Language</option>
+                           <option value="AF">Afrikaans</option>
```

Comment:
This seems like a better thing to put a list of values in javascript and generate the HTML instead of placing it all here.

Reply:
That makes sense, I'll get right on that!

Reply:
Alright, it's automated now!